parquet/src/file/metadata.rs

```diff
@@ -277,6 +298,8 @@ impl RowGroupMetaData {
     }
 
     /// Returns reference of page offset index of all column in this row group.
+    ///
+    /// The returned vector contains `page_offset[column_number][row_group_number]`
```

Comment:
Hmm, I think the returned vector is page offset index for `[column_number][page_number]` in this row group?

Reply:
🤦  yes you are correct

```suggestion
    /// The returned vector contains `page_offset[column_number][page_number]`
```